functions/venue.js

```diff
@@ -586,6 +597,7 @@ exports.toggleDustStorm = functions.https.onCall(async (_data, context) => {
 
 // @debt this is almost a line for line duplicate of exports.updateVenue_v2, we should de-duplicate/DRY these up
 exports.updateVenue = functions.https.onCall(async (data, context) => {
+  console.log("asdasdasd");
```

Comment:
```suggestion
```

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0e1ea16c81870c898660c5e5b60aa9a143b57c80